content/guides/integrations/dropwizard.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Dropwizard"
+date: 2018-11-05T14:42:17-07:00
+draft: false
+class: "shadowed-image lightbox"
+aliases: [/integrations/dropwizard]
+weight: 5
+---
+
+- [Background](#background)
+- [Prerequisities](#prerequisites)
+- [Adding dependencies](#adding-dependencies)
+- [Enable an exporter](#enable-an-exporter)
+- [End to end example](#end-to-end-example)
+- [Visuals](#visuals)
+
```

Comment:
Just one last request, please add a section after Visuals, called References in which you'll then make a table with the various links e.g.
```
### References
Resource|URL
---|---
Dropwizard Project|https://www.dropwizard.io
OpenCensus-Dropwizard on Maven Central|https://mvnrepository.com/artifact/io.opencensus/opencensus-contrib-dropwizard
```
and any other resources